src/Avalonia.DesignerSupport/Remote/PreviewerWindowImpl.cs

```diff
@@ -153,5 +153,7 @@ public void SetExtendClientAreaChromeHints(ExtendClientAreaChromeHints hints)
         public void SetExtendClientAreaTitleBarHeightHint(double titleBarHeight)
         {            
         }
+
+        public void GetWindowsZOrder(Span<Window> windows, Span<long> zOrder) => throw new NotSupportedException();
```

Comment:
I am wondering, if we should use headless windowing subsystem in our previewer code.